examples/multiclass/plot_multiclass_overview.py

```diff
@@ -0,0 +1,200 @@
+"""
+===============================================
+Overview of multiclass training meta-estimators
+===============================================
+
+In this example, we discuss the problem of classification when the target
+variable is composed of more than two classes. This is called multiclass
+classification.
+
+In scikit-learn, all estimators support multiclass classification out of the
+box: the most sensible strategy was implemented for the end-user. The
+:mod:`sklearn.multiclass` module implements various strategies that one can use
+for experimenting or developing third-party estimators that only support binary
+classification.
+
+:mod:`sklearn.multiclass` includes OvO/OvR strategies used to train a
+multiclass classifier by fitting a set of binary classifiers (the
+:class:`~sklearn.multiclass.OneVsOneClassifier` and
+:class:`~sklearn.multiclass.OneVsRestClassifier` meta-estimators). This example
+will review them.
+"""
+
+# %%
+# The Yeast UCI dataset
+# ---------------------
+#
+# In this example, we use a UCI dataset [1]_, generally referred as the Yeast
+# dataset. We use the :func:`sklearn.datasets.fetch_openml` function to load
+# the dataset from OpenML.
+from sklearn.datasets import fetch_openml
+
+X, y = fetch_openml(data_id=181, as_frame=True, return_X_y=True, parser="pandas")
+
+# %%
+# To know the type of data science problem we are dealing with, we can check
+# the target for which we want to build a predictive model.
+y.value_counts().sort_index()
+
+# %%
+# We see that the target is discrete and composed of 10 classes. We therefore
+# deal with a multiclass classification problem.
+#
+# Strategies comparison
+# ---------------------
+#
+# In the following experiment, we use a
+# :class:`~sklearn.tree.DecisionTreeClassifier` and a
+# :class:`~sklearn.model_selection.RepeatedStratifiedKFold` cross-validation
+# with 3 splits and 5 repetitions.
+#
+# We compare the following strategies:
+#
+# * :class:`~sklearn.tree.DecisionTreeClassifier` which supports multiclass
+#   classification by default. The sequential partitioning of the training
+#   dataset isolates data point and affect to the most frequent class
+#   present in the partition. Thus, the decision tree is able to deal with
+#   multiclass classification.
```

Comment:
Just a final nit, I think we can make this paragraph easier to read by wording something similar to:
>:class:`~sklearn.tree.DecisionTreeClassifier` can handle multiclass classification without needing any special adjustments. It works by breaking down the training data into smaller subsets and focusing on the most common class in each subset. By repeating this process, the model can accurately classify input data into multiple different classes.